bot.py

```diff
@@ -1,50 +1,68 @@
+    thread.create_comment(
+        text
+        + "\nBeep Boop! (I'm a bot, and this action was performed automagically.)\nScan timestamp: `"
+        + datetime.now().isoformat()
+        + "`"
+    )
+
 
-targets = ['crystal-linux/.github', 'crystal-linux-packages/.github']
+targets = ["crystal-linux/.github", "crystal-linux-packages/.github"]
 
 status_text = ""
 
 for tgt in targets:
     repo = g.get_repo(tgt)
     status_text += f"# Examining {repo.full_name}\n"
 
-    core_team = g.get_organization('crystal-linux').get_team_by_slug('core-team')
+    core_team = g.get_organization("crystal-linux").get_team_by_slug("core-team")
+    trusted_contrib = g.get_organization("crystal-linux").get_team_by_slug(
+        "trusted-contribututors"
+    )
     core_team_members = core_team.get_members()
-    core_team_logins = []
+    trusted_contrib_members = trusted_contrib.get_members()
+
+    crystal_logins = []
 
     for member in core_team_members:
-        core_team_logins.append(member.login)
+        crystal_logins.append(member.login)
+
+    for member in trusted_contrib_members:
+        crystal_logins.append(member.login)
 
     rfcs_passed = 0
     for issue in repo.get_issues():
-        if 'RFC' in issue.title:
+        if "RFC" in issue.title:
             print(f"Working on: {issue.title}")
             all_reactions = issue.get_reactions()
             if all_reactions.totalCount != 0:
                 total_votes_for = 0
                 total_votes_against = 0
                 for reaction in all_reactions:
-                    if reaction.user.login in core_team_logins:
-                        #print(f"- {reaction.content}: by {reaction.user.login}")
+                    if reaction.user.login in crystal_logins:
+                        # print(f"- {reaction.content}: by {reaction.user.login}")
                         if reaction.content == "+1":
                             total_votes_for += 1
                         elif reaction.content == "-1":
                             total_votes_against += 1
                 print(f"Total votes for resolution: {str(total_votes_for)}")
                 print(f"Total votes against resolution: {str(total_votes_against)}")
-                if (total_votes_for/len(core_team_logins)) > 0.5:
+                if (total_votes_for / len(crystal_logins)) > 0.5:
                     print("This RFC should be passed. Adding comment.")
                     if not issue.locked:
-                        issue.create_comment("This resolution has enough votes from Core Team mebers to be considered passed. Please close the issue once appropriate actions have been taken. - Beep Boop (I'm a bot, and this action was performed automagically)")
+                        issue.create_comment(
+                            "This resolution has enough votes from Core Team members to be considered passed. "
```

Comment:
Nice work, can you just change this to "enough votes" :sweat_smile: then we'll send it!

Reply:
it seems like its already at "enough votes"

Reply:
mb head empty i meant remove "from Core Team members" lol

Reply:
ok did that